api/v1/_submissions/PKPBackendSubmissionsHandler.inc.php

```diff
@@ -75,14 +75,16 @@ public function getSubmissions($slimRequest, $response, $args) {
 			'offset' => 0,
 		);
 
-		$params = array_merge($defaultParams, $slimRequest->getQueryParams());
-
 		// Anyone not a manager or site admin can only access their assigned
 		// submissions
-		if (!$currentUser->hasRole(array(ROLE_ID_MANAGER, ROLE_ID_SITE_ADMIN), $context->getId())) {
+		$userRoles = $this->getAuthorizedContextObject(ASSOC_TYPE_USER_ROLES);
```

Comment:
Where is this ASSOC_TYPE_USER_ROLES set -- I see it is only set in UserRolesRequiredPolicy, but where is this policy called for this and the Dashboard handler below?

Reply:
Ah, I see in the PKPHandler class :+1: